lib/bolt/config.rb

```diff
@@ -247,6 +247,8 @@ def modulepath
     end
 
     def validate
+      Bolt::Util.check_path_case('modulepath', @modulepath)
```

Comment:
Do we want to do this for other configured paths as well, ie. boltdir and inventroyfile?

Reply:
Oddly enough boltdir and inventoryfile don't _seem_ to be case-sensitive.